package.json

```diff
@@ -252,7 +255,7 @@
 				"build": "xcodebuild -workspace ios/RocketChatRN.xcworkspace -scheme RocketChatRN -configuration Debug -sdk iphonesimulator -arch x86_64 -derivedDataPath ios/build",
 				"type": "ios.simulator",
 				"device": {
-					"type": "iPhone 11 Pro"
+					"name": "iPhone_12_Pro_15.5"
```

Comment:
This is not needed

Reply:
Fixed, the name is how I select the device to test.